src/main/java/com/aws/greengrass/clientdevices/auth/CertificateManager.java

```diff
@@ -24,7 +25,7 @@
 import com.aws.greengrass.clientdevices.auth.exception.CloudServiceInteractionException;
 import com.aws.greengrass.clientdevices.auth.exception.InvalidCertificateAuthorityException;
 import com.aws.greengrass.clientdevices.auth.exception.InvalidConfigurationException;
-import com.aws.greengrass.clientdevices.auth.metrics.ClientDeviceAuthMetrics;
+import com.aws.greengrass.clientdevices.auth.metrics.events.CertificateSubscriptionEvent;
```

Comment:
This is a certificate domain event - not a metric. Should be moved into `certificates/events`